src/pb_stub.cc

```diff
@@ -846,11 +846,19 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
   py::class_<InferRequest, std::shared_ptr<InferRequest>>(
       module, "InferenceRequest")
       .def(
-          py::init<
-              const std::string&, uint64_t,
-              const std::vector<std::shared_ptr<PbTensor>>&,
-              const std::set<std::string>&, const std::string&, const int64_t,
-              const uint32_t>(),
+          py::init([](const std::string& request_id, uint64_t correlation_id,
+                      const std::vector<std::shared_ptr<PbTensor>>& inputs,
+                      const std::vector<std::string>& requested_output_names,
```

Comment:
Is requested_output_names supposed to be a set? Or intentionally made a vector of strings in this case?

Reply:
> Is requested_output_names supposed to be a set?

Yes, we intentionally made it a vector so that it works with Python lists.